semantic-conventions/src/tests/semconv/templating/test_markdown.py

```diff
@@ -23,522 +25,139 @@
 
 class TestCorrectMarkdown(unittest.TestCase):
     def testRef(self):
-        semconv = SemanticConventionSet(debug=False)
-        semconv.parse(self.load_file("markdown/ref/general.yaml"))
-        semconv.parse(self.load_file("markdown/ref/rpc.yaml"))
-        semconv.finish()
-        self.assertEqual(len(semconv.models), 5)
```

Comment:
We lose these assertions, but I think they are not very helpful anyway. This test suite is supposed to cover markdown rendering, not model parsing.